CSVSource/src/main/java/io/vantiq/extsrc/CSVSource/CSVReader.java

```diff
@@ -0,0 +1,133 @@
+package io.vantiq.extsrc.CSVSource;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import io.vantiq.extjsdk.ExtensionWebSocketClient;
+/**
+ * Class responsible for the conversion of a line to a relevant json buffer .
+ * its assign the name of the json attributes based on the schema object by compering the offset of the field 
+ * in the line to the value of the schema field attributes name ( field0 , field1 etc). 
+ * if it doesn't find any match it just used those default vakues as the attribute name in the 
+ * new jsn buffer . 
+ * 
+ * each file contnet can be sent to Vatiq using multiple messages , based on the numLinesInEvent 
+ */
+public class CSVReader {
+    private static final String MAX_LINES_IN_EVENT = "maxLinesInEvent";
+    public static ArrayList segmentList = new ArrayList<>();
+/**
```

Comment:
empty line before & indent to same level as code

Reply:
done